contrib/resources/translations/README.md

```diff
@@ -0,0 +1,79 @@
+
+# General translation workflow
+
+## Preparing file with English strings
+
+First, you need up to date the file with English strings. The one present in
+this directory is most likely outdated; to produce the current one do the following:
+
+- In the configuration file, go to section `[dosbox]` and set`language = en`.
+- Run the most current version of DOSBox Staging.
+- Inside DOSBox execute command `config -wl en.txt` (you can supply host OS path
+  instead of bare `en.txt`).
+
+This will produce file with current strings to translate, use it as a reference.
+
+## Updating the translation
+
+To update translation just edit the relevant `*.lng` file with text editor,
+save the file as UTF-8 encoded, with Unix line endings.
+
+Depending on your host OS, use the correct script to normalize the output. Although
+DOSBox Staging should be able to handle non-normalized translation files, and
+normally text editors save normalized files already, it is still highly recommended
+to use the script to make sure no problem occurs.
+
+## String-specific tips
+
+### Format strings
+
+Many strings contain strange sequences starting from percent sign, like `%s` or `%02d`.
+These are so-called _printf format strings_, used by C/C++ programming languages;
+they are replaced with another string or number, using a given format.
+
+You might need to understand the notation to properly format the output (like tables,
+when the translated headers have different length than in English). Search the web
+to learn more about it, here are few links:
+
+- [article on Wikipedia](https://en.wikipedia.org/wiki/Printf)
+- [article on cplusplus.com](https://cplusplus.com/reference/cstdio/printf)
+- [article on cppreference.com](https://en.cppreference.com/w/cpp/io/c/fprintf)
+
+### Configuration option descriptions
+
+For example `CONFIG_FULLSCREEN`.
+
+Do not exceed 79 character in a line, or commands like `config -h fullscreen`
+won't be able display the help properly. The limit might be increased to 80
+characters in the future, but the implementation is not there yet.
+
+### Command or program help
+
+For example `PROGRAM_MOUNT_HELP_LONG` or `SHELL_CMD_CLS_HELP_LONG`. Sometimes
+the first line is extracted to another string, like `SHELL_CMD_CLS_HELP`.
+
+Do not exceed 80 character in a line, or commands like `cls /?` won't be able to
+display the help properly. Also test that command `help /all` properly extracts
+and displays the shortened command descriptions.
```

Comment:
Also test that the`help /all` command properly extracts and displays the shortened command descriptions.

Reply:
OK, changed.